reference/tax_classes.v2.yml

```diff
@@ -142,6 +158,18 @@ definitions:
         description: The name of the tax class.
         example: Shipping
         type: string
+      created_at:
+        description: |
```

Comment:
As far as I can tell we only need this `|` here when there is a multi-line description, but both of these descriptions can be single line.

Or perhaps I don't have the right understanding - thoughts?

Reply:
Easy enough to just make it single line indeed.